edge_multi_trigger.go

```diff
@@ -316,9 +318,11 @@ func (s *EMTState) edgeMultiComputeRecordSpecs(raw []RawType, frameIndexOfraw0 F
 			break
 		}
 		t, u, v = u, v, FrameIndex(x.triggerInd)+frameIndexOfraw0
-		// fmt.Println(t, u, v)
 		recordSpec, valid := edgeMultiShouldRecord(t, u, v, s.npre, s.nsamp, s.mode)
 		if valid {
+			// if (recordSpec.firstRisingFrameIndex - frameIndexOfraw0) < FrameIndex(s.npre) {
+			// 	panic("wtf")
+			// }
```

Comment:
There are several places like this one, where you added but commented out some code. Probably should remove it all.